docs/build-mvm/how-to/schedule_functions.md

```diff
@@ -1 +1,885 @@
+    Connected
+    dealName (host_id: 12D3KooWFS4WXar3f1SWCykTUy9cVKNU8x1yDA18ZYp86mXTUyAe, worker_id: 12D3KooWRRyM8XNhCFTQaqMQpg483Bq8pdfveeMKNxx4hArrNq7u, spell_id: 275382d7-cb42-402c-ad80-85a546ecfe51):
+    
+    2024-02-05 15:36:22 Spell 'myFirstSpell' is working!
+    ```
+    
+    You can also check how the process of installing your spell is happening by checking the `worker-spell` logs with `fluence deal logs`
+    
+    Example output:
+    
+    ```bash
+    $ fluence deal logs
+    Connecting to random local relay: /ip4/127.0.0.1/tcp/9993/ws/p2p/12D3KooWDd7zwsdYart7z9MQ2yEagjafPHLxbzNkNP7wYxJxvKiN
+    Connected
+    dealName (host_id: 12D3KooWFS4WXar3f1SWCykTUy9cVKNU8x1yDA18ZYp86mXTUyAe, worker_id: 12D3KooWRRyM8XNhCFTQaqMQpg483Bq8pdfveeMKNxx4hArrNq7u, spell_id: 4f35fd35-6be3-4a0b-8426-c0f9031d1cfa):
+    
+    2024-02-05 15:36:16 Installing worker for deal 0xeb92a1b5c10ad7bfdcaf23cb7dda9ea062cd07e8
+    2024-02-05 15:36:16 parsed worker definition bafkreifgq24izy6wu22dt6wbrentfcjkmyunof32ospc3qzlkl4adg6tcm {
+      "services": [],
+      "spells": [
+        {
+          "config": "bafkreibk6jbl6qwrp2hktbjkzfctfsyeblffrt3pcm5kte367pes5inhpm",
+          "init_args": "bafkreicecnx2gvntm6fbcrvnc336qze6st5u7qq7457igegamd3bzkx7ri",
+          "name": "myFirstSpell",
+          "script": "bafkreiddhrqqyxhfuyg3egfh34lkhktjplzxjmbmkm22kfjpviory7pjbi"
+        }
+      ]
+    }
+    2024-02-05 15:36:18 Deploying spell myFirstSpell
+    2024-02-05 15:36:22 Installed spell myFirstSpell 275382d7-cb42-402c-ad80-85a546ecfe51
+    2024-02-05 15:36:23 Installation finished
+    2024-02-05 15:36:23 Worker installation finished with status {
+      "message": "",
+      "state": "INSTALLATION_SUCCESSFUL",
+      "timestamp": 1707147383
+    }
+    ```
+    
+
+# Spell Configuration in Fluence CLI
+
+## Spell Triggers Configuration
+
+The `spell.yaml` file allows you to set up the *Spell Trigger,* aka how and on what conditions your spells will be run.
+
+**Timer-based** **Spell Trigger** provides an opportunity to trigger spells at a certain time with a certain regularity or end execution at a certain date. 
+
+To setup a timer-based spell trigger for your spells, you can use the following fields in your `spell.yaml` file:
+
+- `startDelaySec` and `startTimestamp` define when to run the spell (you can use only one of them),
+- `endDelaySec` and `endTimestamp` define when to stop running the spell (you can use only one of them),
+- `periodSec` defines how often to run the spell.
+
+For example, if you want to setup a spell that runs only **once**, you can use the following setup:
+
+```yaml
+clock:
+	periodSec: 0
+```
+
+On the other hand, if you want your spell to run **every two minutes** **indefinitely**, you can do:
+
+```yaml
+clock:
+   periodSec: 120
+```
+
+Next, if we want the spell to **start on New Year’s Eve and end in a year**, we can add the date:
+
+```yaml
+clock:
+   periodSec: 120
+	 startTimestamp: 2024-12-31
```

Comment:
somthing wrong with indentation here. I would recommend just using spaces everywhere in the file